lighthouse-core/gather/driver.js

```diff
@@ -46,11 +46,17 @@ class Driver {
     this._devtoolsLog = new DevtoolsLog(/^(Page|Network)\./);
     connection.on('notification', event => {
       this._devtoolsLog.record(event);
-      this.recordNetworkEvent(event.method, event.params);
+      this._recordNetworkEvent(event.method, event.params);
       this._eventEmitter.emit(event.method, event.params);
     });
     this.online = true;
     this._domainEnabledCounts = new Map();
+
+    /**
+     * Used for monitoring network status events.
+     * @private {?NetworkRecorder}
+     */
+    this._networkStatus = null;
```

Comment:
how about `_networkStatusRecorder`?, *just* networkStatus makes me think it's literally a string 'offline' or something